<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cppQTFilterAppClass</class>
 <widget class="QDialog" name="cppQTFilterAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>693</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Kalman Filter UI</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QChartView" name="gvChart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Generator Setup</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leAmplitude">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>f</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leFrequency">
              <property name="text">
               <string>10</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbSigGenType">
              <item>
               <property name="text">
                <string>Sin</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bin</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Saw</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Buffer Volume</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leBufferVol">
              <property name="text">
               <string>100</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chMakeNoise">
              <property name="text">
               <string>Apply Noise</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Noise A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leNoiseAmplitude">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chFixedStep">
              <property name="text">
               <string>Fixed Generator Step</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Step Value</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leGenStepVal">
              <property name="text">
               <string>0.001</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnApplyGeneratorSettings">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Error Evaluation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Absolute Error Last</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leAbsErrLast">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Relative Error Last (%)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leRelErrLast"/>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Absolute Error Average</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leAbsErrAvg">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Relative Error Average(%)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leRelErrAvg"/>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Average Quadratic Deviation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="leAvgSqDev">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cppQTFilterApp.qrc"/>
 </resources>
 <connections/>
</ui>
